oci/__init__.py

```diff
@@ -8,6 +8,7 @@
 import tarfile
 import zlib
 
+import aiohttp
```

Comment:
as aiohttp is a new dependency, which I would like to avoid, esp. in oci-package, can we also make it optional?
(ignore import-errors + raise runtime-error in async functions); imports for typehints should be possible to be worked around by converting them to str-literatls